src/test/java/com/ksm/wstbackoffice/config/ConfigBean.java

```diff
@@ -0,0 +1,19 @@
+package com.ksm.wstbackoffice.config;
+
+import com.ksm.wstbackoffice.controller.AddressControllerTest;
+import com.ksm.wstbackoffice.controller.CountryControllerTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ConfigBean {
```

Comment:
I inject countryControllerTest, addressControllerTest  in class WstBackofficeApplicationTests,
if you comment two beans in class ConfigBean you will see message -> Could not autowire. No beans of 'CountryControllerTest'. It is reason why I created beans.

Reply:
Why do you need to autowire them in the first place?

Reply:
I tried through constructor but recived rows another errors because chose autowire.
I think chose bad way, I did some change.
Removed code from WstBackofficeApplicationTests class changed AddressControllerTest and CountryControllerTest